src/avfoundation.cs

```diff
@@ -2358,6 +2368,13 @@ interface AVAudioSession {
 		[Export ("inputOrientation")]
 		AVAudioStereoOrientation InputOrientation { get; }
 
+		[Watch (7,3), TV (14,5), NoMac, iOS (14,5)]
```

Comment:
@rachelkang you're the one who told me it's 7.4 (not 7.3) for Xcode 12.5 ;-)
Do a find-n-replace (there's likely more than one)

Reply:
oops, thanks! 😂